### ISSUE TITLE
Create a "Definition of Done" page

### ISSUE THREAD

**Issue by annethyme** [auto-wcag/auto-wcag#241]
**Note:** Suggesting a new page that we can link to from the "Contribute" page:

# Definition of Done

- The relation between the rule and its accessibility requirements is as described in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/) under [Accessibility Requirements](https://www.w3.org/TR/act-rules-format/#structure-accessibility-requirements). This includes:
    - The rule is an appropriate test for the accessibility requirement it sets out to test, e.g. from [WCAG 2](https://www.w3.org/TR/WCAG21/), and in line with the supporting documents, e.g. [Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/) and [Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG-TECHS/)
    - Failing the rule will for sure mean failing the accessibility requirements (success criteria) linked to under the "Success criteria" heading. If this is not true, the success criteria can instead be mentioned under the heading "Background".
    - Failing this rule will not mean failing other accessibility requirements (success criteria) than the ones linked to from this rule. If this was the case, these accessibility requirements needed to be added.
    - The rule allows for solutions both through native markup languages and other technologies such as WAI-ARIA
- Applicability (for [atomic](https://www.w3.org/TR/act-rules-format/#test-applicability) / [composed rules](https://www.w3.org/TR/act-rules-format/#aggregation-applicability)) and Expectations (for [atomic](https://www.w3.org/TR/act-rules-format/#test-expectations) / [composed rules](https://www.w3.org/TR/act-rules-format/#aggregation-expectations)) live up to the requirements for these sections in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/)
- Requirements for use of atomic and composed rules are followed, see [Rule Types](https://www.w3.org/TR/act-rules-format/#rule-types) in the [ACT Rules Format](https://www.w3.org/TR/act-rules-format/)
- The rule follows the Auto-WCAG [rule template](/design/rule-template.html), especially in relation to headings, styling, test case descriptions, etc.
- The rule is using Auto-WCAG [Glossary terms](https://auto-wcag.github.io/auto-wcag/pages/algorithms/) whenever possible. Be particularly aware of the following much-used algorithms:
    - For the Applicability, consider if the definitions [exposed to assistive technologies](#exposed-to-assistive-technologies) and [visible on the page](#visible-on-the-page) should be used to narrow down the scope of the rule.
    - For the Applicability and Expectations, consider if the definition for [semantic role](#semantic-role) (including specifics of explicit and implicit semantic role) could be used to describe the targets of the rule.
- The rule links to any relevant documentation, e.g. [Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/) and [Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG-TECHS/), specifications used, etc.
- The rule has been spellchecked
- All links are working, or the linked files are included in the same pull request

**Comment by annethyme** [comment]
Probably better with a Definition of Done, that can be used both by rule authors and reviewers than just a checklist for reviewers as originally proposed.
